Jenkinsfile

```diff
@@ -78,7 +78,8 @@ def job = {
         config.docker_tag = config.ksql_db_version
     } else {
         config.docker_tag = config.ksql_db_version.tokenize("-")[0] + '-' + env.BUILD_NUMBER
-        config.revision = 'refs/heads/master'
+        // Use revision param if provided, otherwise default to master
+        config.revision = params.GIT_REVISION ?: 'refs/heads/master'
```

Comment:
we can move this line to be after the else statement now, and remove line 75.

Reply:
Thanks for the suggestion, @elismaga. Done in 7874f2d